packages/@coorpacademy-progression-engine/src/types.ts

```diff
@@ -82,15 +82,15 @@ export interface ResourceContent {
 export type Content = GenericContent | ResourceContent;
 
 export interface ViewedResource {
-  type: $PropertyType<Content, 'type'>,
+  type: Pick<Content, 'type'>,
   ref: string,
   resources: Array<string>
 };
 
 export interface AnswerRecord {
   slideRef: string,
   answer: Answer,
-  isCorrect: ?boolean
+  isCorrect: boolean | void
```

Comment:
incorrect, should be: `isCorrect?: boolean | null`

Reply:
non, c'est bien void